src/cammainwindow.cpp

```diff
@@ -709,31 +715,9 @@ void CamMainWindow::updateCurrentSettingsLabel()
 	} else {
 		/* Show secondary exposure period as shutter angle. */
 		int shutterAngle = (expPeriod * 360.0) / framePeriod;
-		sprintf(shString, "%u\xb0", max(shutterAngle, 1)); /* Round up if less than zero degrees. */
+        sprintf(shString, "%u\xb0", max(shutterAngle, 1)); /* Round up if less than zero degrees. */
```

Comment:
In the future, please keep the spacing vs tabs consistent with the rest of the codebase.